plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -452,12 +453,24 @@ export class DeploymentManager {
       if (typeof first === 'string') {
         debug(`[${this.network}] ${first}`, ...rest);
       } else {
-        return first.wait().then(async (tx) => {
-          const cost = Number(txCost(tx) / (10n ** 12n)) / 1e6;
-          const logs = tx.events.map(e => `${e.event ?? 'unknown'}(${e.args ?? '?'})`).join(' ');
+        return first.wait().then(async (tx: ContractReceipt) => {
+          const cost = Number(txCost(tx) / 10n ** 12n) / 1e6;
+          const logs  = tx.events.map((evt) => {
+            let abi = [
+              'event ProposalCreated(uint256,address,address[],uint256[],string[],bytes[],uint256,uint256,string)',
+              'function Propose(uint256, address, address[],uint256[],string[],bytes[], uint256, uint256, string)'];
+            let iface = new ethers.utils.Interface(abi);
+            try {
+              let decoded = iface.parseLog(evt);
+              let calldata = iface.encodeFunctionData('Propose', decoded.args);
+              debug(['calldata:', calldata].join(' '));
```

Comment:
```suggestion
              if('ProposalCreated' === evt.event) 
                (({to,data})=>debug(`to: ${to} calldata: ${data}`))(tx.getTransaction());
```
* Would something like this work?  
* What is the output of this suggestion here?
* Could this replace the other added lines?
* Is this preferred for locally simulating the transaction that emitted the event?
[Reference](https://docs.ethers.org/v5/api/contract/contract/#contract-functionsSend)

Reply:
Evt.event is less reliable structure, it well might occur as "undefined" and depends on migration script implementation.

Reply:
Does it work for any of the existing migrations?  Please provide a sample of the existing migration script implementations that don't work.

Use exception handling logic for branching as a last result like if event name is undefined. don't output debug message that the event type is unknown if only handling proposalcreated.